test/manual/amp-consent/legacy_config_support/amp-consent-cmp-screen-reader.html

```diff
@@ -0,0 +1,392 @@
+<!doctype html>
```

Comment:
Please don't add examples to `legacy_config_support` folder as they are meant to verify the deprecated the config format. 
Also I think we can combine the `captions` config into one of the existing example instead of creating a new one.